samples/SocialSample/Startup.cs

```diff
@@ -58,6 +61,17 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerfactory)
             {
                 options.ClientId = "560027070069-37ldt4kfuohhu3m495hk2j4pjp92d382.apps.googleusercontent.com";
                 options.ClientSecret = "n2Q-GEw9RQjzcRbU3qhfTj8f";
+                options.Events = new OAuthEvents()
+                {
+                    OnRemoteError = ctx =>
```

Comment:
Hum, understood. Are there plans to make this error path better in the future? Looks like throwing exceptions and letting them surface up to the (optional) exception handler middleware might be a better approach.